### ISSUE TITLE
[client] Accessibility: Announce the status of search results

### ISSUE THREAD

**Issue by chrisshaw** [hypothesis/product-backlog#1349]
**As a user using assistive technologies like NVDA, JAWS, or VoiceOver, I want to be notified of the status of search results, so I know to scan the results to engage in discussions more quickly.**

A reviewer in an accessibility review considered it a major defect that, upon searching for annotations, the user would not know anything actually happened:

> The sidebar displayed “Showing 2 results for ‘work’,” but the screen reader did not hear anything.
> 
> (Also had to skip over the expanded Help text.)
> 
> Use alert notification showing 2 results; Provide a way to skip to the results.
> 
> Major

From our WCAG audit, the recommended solution is a status role:

> Searching annotations results in a status message that does not have an ARIA status role.
> 
> ![image](https://user-images.githubusercontent.com/2966918/165684602-d4f1669a-0943-4723-942c-86d751592ef4.png)
> _Search results message_
> 
> ### Recommendation for status messages
> Add a role=”status” to the search results message.

**Comment by chrisshaw** [comment]
**Issue was linked to Harvestr Discovery:**  [Hear announcements for search results via screen reader](https://app.harvestr.io/components/0/list/b76ssZEdg)

**Comment by lyzadanger** [comment]
Cool. ARIA roles are doing and more and more heavy lifting these days.
